server.js

```diff
@@ -175,7 +171,7 @@ app.put('/posts/:id', (req, res) => {
         res.status(400).json({ message: message });
     }
 
-    const toUpdate = {};
+    let toUpdate = {}; // you change it later, const means no change - removed.
```

Comment:
the `toUpdate` variable is used later -- a const would make it impossible to change.